python/perspective/perspective/__init__.py

```diff
@@ -16,4 +16,8 @@
 from .client import *
 from .handlers import *
 from .nbextension import _jupyter_nbextension_paths
-from .widget import *
+from .widget.widget import PerspectiveWidget
```

Comment:
this is an affordance to make `%autoreload` magic work better. happy to revert

Reply:
We need to go through and make sure all our * imports pull from files with `__all__`, this is a good first issue I'll put in another ticket

Reply:
https://github.com/finos/perspective/issues/2419

Reply:
reverted this